### ISSUE TITLE
blank page after login with PTC

### ISSUE THREAD

**Issue by fbrcc** [billyvg/pokemon-journal#28]
I've been having this issue since yesterday, today I pulled the last package.json update. It does recognize wrong PTC passwords as I only get the blank screen when I enter the correct one. Thanks.

**Comment by ghost** [comment]
I can confirm that the blank page issue persists after the package.json update. Using Google login here.

**Comment by billyvg** [comment]
Did you run `npm install` after pulling?

**Comment by ghost** [comment]
I'm an idiot. I actually didn't pull because I've made some changes of my own to my local copy of the project and simply edited pogobuf in package.json. It's working after re-running `npm install`. Thanks.

**Comment by Mawnstah** [comment]
Alright I know this is a noob question, but I downloaded the pre-built package. And I am also getting the blank screen error. I found the package.json file on in the folders but I'm not sure what to do with it on my end in order for the pokemon journal to work again. It was working fine yesterday, can anyone point me in the right direction?

BTW I don't have python installed or anything like that, and I'm not too familiar with it hence the reason I downloaded the pre-built package.

**Comment by fbrcc** [comment]
It's working after running npm install. Thanks

**Comment by Alkene** [comment]
Could you link to a guide on how to compile and run this or update the pre-made packages please?  THanks

**Comment by aisbergde** [comment]
How can I install the latest version?
If I execute
`npm install -g pokemon-journal`
it looks like a version pokemon-journal@0.1.2 is trying to be installed and I get errors
I don't know how npm works and why this old version 0.1.2 is used.

If on the other hand I get the latest version from github into a local folder this version looks like pokemon-journal@0.2.0 but I don't know how to use it:
`D:\Git_Repositories\pokemon-journal>npm run dev`
produces errors I described before:

> pokemon-journal@0.2.0 dev: `concurrently --kill-others "npm run hot-server" "npm run start-hot"`

In other words: it looks like there is a version 0.2.0, but how to compile or execute it?

**Comment by billyvg** [comment]
New release here https://github.com/billyvg/pokemon-journal/releases/tag/0.2.1

**Comment by Mawnstah** [comment]
Thanks for compiling the new version man. Much appreciated.
